api/fixtures/example.go

```diff
@@ -29,6 +30,7 @@ func (o *ExampleResources) AsObjects() []crclient.Object {
 		o.KubeCloudControllerAWSCreds,
 		o.NodePoolManagementAWSCreds,
 		o.Cluster,
+		o.NodePool,
```

Comment:
I theorize if you run with `--node-pool-replicas=0` the program will crash due to this array element being `nil`... Some day I'll refactor this fixture thing...